Lib/test/test_contextlib.py

```diff
@@ -152,6 +152,36 @@ def woohoo():
         else:
             self.fail('StopIteration was suppressed')
 
+    def test_contextmanager_do_not_unchain_non_stopiteration_exceptions(self):
+        code = """\
+from contextlib import contextmanager
+@contextmanager
+def test_issue29692():
+    try:
+        yield
+    except Exception as exc:
+        raise RuntimeError('issue29692:Chained') from exc
+"""
+        locals = {}
+        exec(code, locals, locals)
```

Comment:
Since `from __future__ import generator_stop` no longer used, the trick with `exec()` no longer needed.